modules/bootstrapped/src/generated/smithy4s/example/PizzaAdminService.scala

```diff
@@ -248,93 +248,93 @@ object PizzaAdminServiceOperation {
     def $ordinal: Int
 
     object project {
-      def genericClientError: Option[GenericClientError] = GetMenuError.GenericClientErrorCase.alt.project.lift(self).map(_.genericClientError)
-      def fallbackError2: Option[FallbackError2] = GetMenuError.FallbackError2Case.alt.project.lift(self).map(_.fallbackError2)
-      def fallbackError: Option[FallbackError] = GetMenuError.FallbackErrorCase.alt.project.lift(self).map(_.fallbackError)
       def notFoundError: Option[NotFoundError] = GetMenuError.NotFoundErrorCase.alt.project.lift(self).map(_.notFoundError)
+      def fallbackError: Option[FallbackError] = GetMenuError.FallbackErrorCase.alt.project.lift(self).map(_.fallbackError)
+      def fallbackError2: Option[FallbackError2] = GetMenuError.FallbackError2Case.alt.project.lift(self).map(_.fallbackError2)
+      def genericClientError: Option[GenericClientError] = GetMenuError.GenericClientErrorCase.alt.project.lift(self).map(_.genericClientError)
     }
```

Comment:
not sure which change triggered this reordering. https://github.com/smithy-lang/smithy/pull/2513 looks related, but it's in the IDL serializer, and resource-specific. I checked the changelog for "sort" and "order" and "properties" but nothing else stood out.

still, this is the order as in the spec.